.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,7 +11,7 @@ https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
 -->
 
 ##### Checklist
-<!-- remove lines that do not apply to you -->
+<!-- Remove items that do not apply, For completed items, change [ ] to [x]. -->
```

Comment:
Maybe remove the period at the end of this? Less potentially confusing, easier to copy.